src/clr-angular/forms/common/label.ts

```diff
@@ -8,12 +8,14 @@ import { Subscription } from 'rxjs';
 
 import { IfErrorService } from './if-error/if-error.service';
 import { ControlIdService } from './providers/control-id.service';
+import { LayoutService } from './providers/layout.service';
 
 @Directive({ selector: 'label' })
```

Comment:
I know this was introduced before but won't the HTML tag selector `label` target all labels whether they are in or outside the form?

Reply:
I think I missed asking about this in the earlier review.

Reply:
Yes they do target all labels, but are largely inert unless they are inside a form control container that provides the services.